templates/zerver/emails/email.css

```diff
@@ -20,19 +21,20 @@ body {
     -webkit-text-size-adjust: 100%;
 }
 
-pre {
-    font-family: sans-serif;
-    font-size: 14px;
-}
-
-table {
+table.body,
+table.main,
+td.wrapper > table,
+.footer > table {
     border-collapse: separate;
     mso-table-lspace: 0;
     mso-table-rspace: 0;
     width: 100%;
 }
 
-table td {
+table.body td,
```

Comment:
CSS nesting is not functional in `email.css`, so I had to manually copy the selectors from above to accurately target the tables we want to apply CSS to. While nesting would be preferable, it would take time to figure out how to make it work, so I decided to copy.

Reply:
The repetition is fine with a comment, but I worry a bit that you may need it to be `> table > tr > td` or something like that -- otherwise, this will apply to `td` elements nested inside one of the tables that are not a direct child of the tables you wanted to apply this to.

Reply:
Now when we have added the "layout" class to all the tables we want to apply CSS to, the process has been simplified to just use `table.layout > tr > td`